desktop/windows/release-notes/index.md

```diff
@@ -23,15 +23,29 @@ This page contains information about the new features, improvements, known issue
 
 Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/projects/1){: target="_blank" rel="noopener" class="_"} to see what's coming next.
 
-## Docker Desktop 4.5.0
-2022-02-10
+## Docker Desktop 4.5.1
+2022-02-14
 
 > Download Docker Desktop
 >
 > [For
 > Windows](https://desktop.docker.com/win/main/amd64/Docker%20Desktop%20Installer.exe?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-win-amd64){:
 > .button .primary-btn }
 
+### Bug fixes and minor changes
+
+- Fixed an issue that caused new installations to default to the Hyper-V backend instead of WSL 2.
+- Fixed a crash in the Docker Dashboard which would make the systray menu disappear.
+
+If you are running Docker Desktop on Windows Home, installing 4.5.1 will switch it back to WSL 2 automatically. If you are running another version of Windows, and you want Docker Desktop to use the WSL 2 backend, you must manually switch by enabling the **Use the WSL 2 based engine** option in the **Settings > General** section.
+
```

Comment:
@stephen-turner suggests we had something like
```
Alternatively you can edit the Docker Desktop settings file and manually switch the value of the `wslEngineEnabled` field to `true`.
```
@usha-mandya any thought?

Reply:
I like that except let's give the actual path name of the file.

Reply:
@mat007 Sounds good. I'd also include the location of the settings file.

"Alternatively, you can edit the Docker Desktop settings file located at `<location of the json file`  and manually switch the value of the `wslEngineEnabled` field to `true`."

Reply:
Ah sorry, I had rephrased it in the meanwhile as well:
> Alternatively with Docker Desktop 4.5.0, you can edit the `%APPDATA%\Docker\settings.json` file and switch the value of the `wslEngineEnabled` field to `true`, then restart Docker Desktop.

Reply:
I have used your version, Usha.